homeassistant/components/edl21/sensor.py

```diff
@@ -53,6 +53,9 @@
     # A=1: Electricity
     # C=0: General purpose objects
     # D=0: Free ID-numbers for utilities
+    # E=0 Ownership ID
+    SensorEntityDescription(key="1-0:0.0.0*255", name="Ownership ID", icon="mdi:flash"),
```

Comment:
Is this value interesting for most users or should we disable it by default?

https://developers.home-assistant.io/docs/entity_registry_disabled_by#integrations-setting-default-value-of-disabled_by-for-new-entity-registry-entries

Reply:
Probably not interesting for most users, I changed the code accordingly.